dockerfiles/ci/xfail_tests/5.4.list

```diff
@@ -257,3 +257,5 @@ ext/curl/tests/curl_setopt_error.phpt
 ext/pdo_sqlite/tests/bug50728.phpt
 ext/pdo/tests/pdo_023.phpt
 ext/pdo_sqlite/tests/pdo_013.phpt
+tests/lang/bug23584.phpt
+Zend/tests/bug48930.phpt
```

Comment:
On PHP 5.4 the shebang throws off the line numbers when `auto_prepend_file` is used. The [bug was fixed in PHP 5.5](https://heap.space/xref/PHP-5.5/main/main.c?r=c117548e#2511-2515).